README.md

```diff
@@ -61,6 +79,81 @@ For a full example of `build.sbt` you can have a look at this [build.sbt](https:
 used for the example.
 
 
+<a name="use-pureconfig" />
+## Use PureConfig
+
+Import the library and use one of the `loadConfig` methods:
+
+
+```scala
+import pureconfig._
+
+val config: Try[YourConfClass] = loadConfig[YourConfClass]
+```
+
+
+<a name="supported-types" />
+## Supported Types
+
+Currently supported types for fields are:
+- `String`, `Boolean`, `Double` (standard
+and percentage format ending with `%`), `Float` (also supporting percentage),
+`Int`, `Long`, `Short`, `URL`, `Duration`
+- all collections implementing the `TraversableOnce` trait where the elements type
+is in this list
+- `Option` for optional values, i.e. value that can or cannot be in the configuration
+- `Map` with `String` keys and any value type that is in this list
+- typesafe `ConfigValue`, `ConfigObject` and `ConfigList`.
+
+
+<a name="extend-the-library-to-support-new-types" />
+## Extend the library to support new types
+
+Not all types are supported automatically by `pureconfig`. For instance, classes
+with mutable fields are not supported out-of-the-box:
```

Comment:
I don't think this is accurate. `pureconfig` supports case classes with mutable fields. I think we can refer to classes that are not case classes in this example.